src/main/vfxutil.js

```diff
@@ -0,0 +1,59 @@
+export let transparency = true;
+
+export function getTransparency(){
+  return transparency;
+}
+export function toggleTransparency(){
+  transparency = !transparency;
+}
+
+export function makeColour(r, g, b, a) {
+  // maybe some hsl too
+  if (transparency) {
+    return "rgba(" + r + "," + g + "," + b + "," + a + ")";
+  }
+  else {
+    return "rgb(" + r + "," + g + "," + b + ")";
```

Comment:
ES6 has string interpolation built in to the language:
```
return `rgba(${r},${g},${b},${a}`;
```
A little more succinct and readable.